PayItForward.ConsoleClient.UnitTests/Web/Controllers/HomeController/Index_Should.cs

```diff
@@ -0,0 +1,192 @@
+            var totalPages = (int)Math.Ceiling(1 / (decimal)3);
+            var skip = (page - 1) * 3;
+            var stories = this.storiesServices.Setup(s => s.GetStories(3, skip, "/Home/Index/", empty))
+                .Returns(this.HelperStoryDto());
+
+            // Act
+            var result = this.homeController.Index(1, empty, "/Home/Index/");
+
+            // Assert
+            this.storiesServices.Verify(x => x.GetStories(3, 0, "/Home/Index/", empty), Times.Once);
+        }
+
+        [Fact]
+        public void CallGetCategoriesOnce()
+        {
+            // Arrange
+            var empty = string.Empty;
+            var page = 1;
+            var totalNumberOfStories = this.storiesServices.Setup(
+                services =>
+                        services.CountStories("/Home/Index/", empty))
+                        .Returns(this.HelperIndexViewModelToTest().Stories.Count());
+            var totalPages = (int)Math.Ceiling(1 / (decimal)3);
+            var skip = (page - 1) * 3;
+            var stories = this.storiesServices.Setup(s => s.GetStories(3, skip, "/Home/Index/", empty))
+                .Returns(this.HelperStoryDto());
+
+            var categories = this.categoriesService.Setup(c => c.GetCategories())
+                .Returns(this.HelperCategoriesListWithCategoryDTO());
+
+            // Act
+            var result = this.homeController.Index(1, empty, "/Home/Index/");
+
+            // Assert
+            this.categoriesService.Verify(x => x.GetCategories(), Times.Once);
+        }
+
+        private IndexViewModel HelperIndexViewModelToTest()
+        {
+            var basicStoryViewModel = this.HelperStoryDto();
+            return new IndexViewModel
+            {
+                CurrentPage = 1,
+                TotalPages = 3,
+                Stories = this.mapper.Object.Map<IEnumerable<BasicStoryViewModel>>(basicStoryViewModel),
+                SearchWord = string.Empty,
+                CurrentUrl = "/Home/Index"
+            };
+        }
+
+        private List<StoryDTO> HelperStoryDto()
+        {
+            return new List<StoryDTO>()
+            {
+                new StoryDTO
+                {
+                    Category = new CategoryDTO()
+                    {
+                        Name = "Health"
+                    },
+                    CollectedAmount = 0,
+                    CreatedOn = DateTime.UtcNow,
+                    Description = "Some description",
+                    Title = "Title",
+                    User = new UserDTO()
+                    {
+                        Email = "vicky.penkova@gmial.com",
+                        FirstName = "Viki",
+                        LastName = "Penkova",
+                        AvilableMoneyAmount = 100
+                    }
+                }
+            };
+        }
+
+        private List<CategoryDTO> HelperCategoriesListWithCategoryDTO()
```

Comment:
HelperCategoriesListWithCategoryDTOcan be renamed to GetTestCategoriesListWithCategoryDTOs because it is method and it should talk about what it is